libraries/AP_TemperatureSensor/TSYS01.h

```diff
@@ -5,7 +5,6 @@
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-#include <AP_HAL/Semaphores.h>
```

Comment:
The .cpp file is making use of semaphores; we really ought to include the header in the .cpp file.